crates/c-api/include/wasmtime/async.h

```diff
@@ -11,22 +11,22 @@
  * are three mechanisms for yielding control from wasm to the caller: fuel,
  * epochs, and async host functions.
  *
- * When WebAssembly is executed, a #wasmtime_call_future_t is returned. This
+ * When WebAssembly is executed, a `wasmtime_call_future_t` is returned. This
```

Comment:
unrelated changes?

Reply:
I found this was necessary at one stage of testing to appease doxygen. That being said I barely understanding doxygen and we could very well be holding doxygen wrong where this should work when it doesn't.